src/views/private/navbar/index.js

```diff
@@ -1,14 +1,43 @@
 import React from "react";
-import { NavLink } from "react-router-dom";
+import { NavLink, useNavigate } from "react-router-dom";
 import { adminRoutes, routes } from "../../../shared/Constant";
+import CustomButton from "../../../components/atoms/customButton";
+import { useDispatch } from "react-redux";
+import { logoutUser } from "../../../redux/action";
+import {toast} from "react-toastify"
+import './style.css'
 export default function Navbar() {
-    console.log(routes.ADMIN+adminRoutes.CREATE_CATEGORY)
+    const dispatch = useDispatch();
+    const navigate = useNavigate();
+    const handleLogout = () => {
+        localStorage.clear()
+        dispatch(logoutUser({handleResponse}))
+    }
+    const handleResponse = (response) => {
+        console.log(response)
+        navigate('/login')
+        if (response.status === 200) {
+
+            toast.success("logout  Successfullly "
+                , {
+                    position: toast.POSITION.TOP_RIGHT,
+                })
+        }
+
+        else {
+            toast.error(response.response.data.message
+                , {
+                    position: toast.POSITION.TOP_RIGHT,
+                })
+        }
+    }
+    
     return (
         <>
-        {/*admin navbar */}
-            <nav className="navbar navbar-expand-lg navbar-light bg-light">
+            {/*admin navbar */}
+            <nav className="navbar navbar-expand-lg navbar-light bg-light navbar_update">
                 <div className="container-fluid">
-                    <a className="navbar-brand" href="#">Navbar</a>
+                    <a className="navbar-brand" href="#"> Hi Admin </a>
```

Comment:
try to use link of react router instead of anchor tag

Reply:
ok if no link then what to Use ???

Reply:
check where its redirction give route of that page as url in link